pkg/commands/cluster/dump/capture/capture.go

```diff
@@ -108,12 +105,12 @@ func CaptureAllResources(cp CaptureParams) error {
 
 // PutIntoNodeNamesIfNotPresent populates the node map with a given node name
 func PutIntoNodeNamesIfNotPresent(inputKey string) {
-	knownNodeNamesMutex.Lock()
-	_, ok := KnownNodeNames[inputKey]
+	sanitize.KnownNodeNamesMutex.Lock()
```

Comment:
if it better to immediately defer an unlock after locking in case someone changes the flow in the future where the function is exited without unlocking

Reply:
It might be better just to move this function to sanitize package which owns the lock.  It sort of feels like it should be there.